bin/cyhy-ip

```diff
@@ -163,6 +162,12 @@ def add(db, owner, cidrs):
     intersections = db.RequestDoc.get_all_intersections(
         cidrs
     )  # intersections from database
+    geo_loc_db = GeoLocDB()
+    for cidr in cidrs.iter_cidrs():
+        if geo_loc_db.check_restricted_cidr(cidr):
+            print("Cannot continue!\nSome addresses are in restricted countries.")
+            sys.exit(-1)
```

Comment:
This won't check every CIDR, Is this desired behavior?

Reply:
This is within a 'for cidr in cidrs.iter_cidrs(): statement. Does that not loop through the cidrs as expected? If not, we can change the logic.

Reply:
When you call `sys.exit(-1)` you will "bail out" at the first restricted CIDR.

Reply:
Ahh yeah.  I will see what I can do to resolve

Reply:
@mcdonnnj, I think this is resolved now.  Do you agree?

Reply:
![image](https://user-images.githubusercontent.com/50747025/111652363-68c20500-87dd-11eb-8b2f-dc71418b3347.png)